### DISCUSSION TITLE
Creating webhooks with the API? "Only apps can subscribe to webhooks"

### DISCUSSION THREAD

**Discussion by danthegoodman1** [vercel/vercel#4664] [Help]
I keep getting the "Only apps can subscribe to webhooks", using test tokens or generated tokens. I'm not sure what I need to do to create an app since there is no mention in the API docs. Does this mean I can only do webhooks to Vercel deployments serverless functions?

Sorry if this is a simple answer and I'm totally glazing over it.
Thanks.

**Comment by IRegiani** [comment]
Also having the same issue, had any luck solving it?

**Comment by danthegoodman1** [comment]
I just modified the build process to use a custom script to get the info I
want, similar results

**Comment by lukasIO** [comment]
having the same issue, a custom script during build time is not an option in my case. 
how can we create a webhook via the api or what is needed in order to do successfully subscribe to a webhook?

**Comment by GoldElysium** [comment]
Same question here

**Comment by morfarpeter** [comment]
Same here... I can't find the right API docs for webhooks, and I don't understand what `only apps` means here? I just need a way to create a webhook for deployment ready on my project/deployment when I create it through the API.

**Reply by kunke32** [reply]
yeah
